pom.xml

```diff
@@ -1236,11 +1244,55 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/org.openclover/clover-maven-plugin -->
+            <dependency>
+                <groupId>org.openclover</groupId>
+                <artifactId>clover-maven-plugin</artifactId>
+                <version>${version.org.openclover}</version>
+            </dependency>
+
         </dependencies>
     </dependencyManagement>
 
     <profiles>
         <profile>
+<!--To generate a report on test coverage run
+mvn clean install -Ptest-coverage org.openclover:clover-maven-plugin:aggregate org.openclover:clover-maven-plugin:clover
```

Comment:
Would be good to add some information on how to view the report (perhaps at the end of the "Setting up your developer environment" section [here](https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md#setting-up-your-developer-environment). There, we mention how to run tests so we could add some info about generating the test coverage report there as well.

Reply:
@fjuma I added some text on the section you suggested about the report generation and how to access it. Check it out and tell me if you think it is Ok now.